charts/piraeus/crds/operator-controllerset-crd.yaml

```diff
@@ -44,6 +44,11 @@ spec:
                 PEM format
               type: string
               nullable: true
+            dbPresentClientCert:
+              description: Present a TLS client certificate for authentication with the database (etcd). If set to true,
+                `dbCertSecret` must be set and contain two additional entries "client.cert" and "client.key" in PEM
+                format.
```

Comment:
Your docs say that one of them should be in PKCS8 format.

Reply:
Updated